handlers/doc.go

```diff
@@ -242,3 +242,17 @@ type singleScheduleResponseWrapper struct {
 	// in: body
 	Body models.Schedule
 }
+
+// Parameters to take the application files
+// swagger:parameters idPostApplicationFile
+type applicationFilesParamsWrapper struct {
+	// in: body
+	ApplicationFile string `json:"application_file"`
+}
+
```

Comment:
@sayantan1413 we can remove this parameter wrapper, I checked and there isn't any specified struct for parameters, so in such case we can leave it empty.

Reply:
Oh yeah sure I will remove it.